embedded_fab/main.dart

```diff
@@ -0,0 +1,416 @@
+              appBar: AppBar(
+                actions: <Widget>[
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 16.0),
+                    child: IconButton(
+                      icon: Icon(Icons.send),
+                      onPressed: () {
+                        Navigator.pop(context);
+                      },
+                    ),
+                  )
+                ],
+                backgroundColor: Colors.white,
+                brightness: Brightness.light,
+                elevation: 0,
+                iconTheme: IconThemeData(color: grey800),
+              ),
+              body: Padding(
+                padding: const EdgeInsets.all(16.0),
+                child: Column(
+                  children: <Widget>[
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'To',
+                      ),
+                    ),
+                    SizedBox(height: 8),
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'Subject',
+                      ),
+                    ),
+                    SizedBox(height: 8),
+                    TextField(
+                      decoration: const InputDecoration(
+                        border: OutlineInputBorder(),
+                        filled: false,
+                        hintText: 'Compose email',
+                      ),
+                      maxLines: 10,
+                    ),
+                    SizedBox(height: 8),
+                  ],
+                ),
+              ),
+              extendBody: true,
+            );
+          },
+          fullscreenDialog: true,
+        ));
+  }
+
+
+  List<Widget> _drawerOptions() {
+    return [
+      Text('Options', style: Theme.of(context).textTheme.display1),
+      Divider(),
+      ListTile(title: Text('Shape')),
+      RadioListTile(
+        title: Text('Flat'),
+        value: ShapeType.Flat,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      RadioListTile(
+        title: Text('Bump'),
+        value: ShapeType.Bump,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      RadioListTile(
+        title: Text('Cut'),
+        value: ShapeType.Cut,
+        groupValue: _selectedShape,
+        onChanged: (value) {
+          _setSelectedTile(value);
+        },
+      ),
+      ListTile(title: Text('FAB Color')),
+      RadioListTile(
+        title: Text('White'),
+        value: Colors.white,
+        groupValue: _selectedColor,
+        onChanged: (value) {
```

Comment:
I think this can just be: onChanged: _setSelectedColorTile

Reply:
Done